src/hooks/useAccountBalance.ts

```diff
@@ -135,12 +152,21 @@ export const useAccountBalance = ({
   const nativeStakingCoinBalanace = stakingBalances?.[chainTokenDenom];
   const nativeStakingBalance = MustBigNumber(nativeStakingCoinBalanace?.amount).toNumber();
 
-  return {
+  const res = {
     balance,
     nativeTokenBalance,
     nativeStakingBalance,
     usdcBalance,
-    queryStatus: isCosmosChain ? cosmosQuery.status : evmNative.status,
-    isQueryFetching: isCosmosChain ? cosmosQuery.isFetching : evmNative.isFetching,
+    queryStatus: isCosmosChain
```

Comment:
[suggestion] there's quite a bit of ternary branching, can we make a simple function or break this into several variable assignments that build on each other?

Reply:
broke out the ternary so its more readable